pkg/ocm/versions.go

```diff
@@ -36,7 +36,7 @@ const (
 	LowestHttpTokensRequiredSupport = "4.11.0"
 	LowestSTSMinor                  = "4.7"
 
-	LowestHostedCpSupport            = "4.14.0"
+	LowestHostedCpSupport            = "4.14.0-0.a"
```

Comment:
We've noticed that this was wrongly introduced.

https://redhat-internal.slack.com/archives/C04C3G5MDKN/p1710502133096589

Reply:
I wonder if we could use this one or another PR to address it.

I'm not sure about the implications, perhaps a separated PR is good.

But I also missed from my side, I didn't have all context.

Reply:
@lucasponce As discussed in Slack, I think I approved this just as you were commenting 😄 . Let's follow up with another PR to fix the 4.12 issue.

cheers,

Rob